runners/spark/src/main/java/org/apache/beam/runners/spark/translation/EvaluationContext.java

```diff
@@ -58,193 +54,137 @@
  */
 public class EvaluationContext implements EvaluationResult {
   private final JavaSparkContext jsc;
-  private final Pipeline pipeline;
+  private JavaStreamingContext jssc;
   private final SparkRuntimeContext runtime;
-  private final Map<PValue, RDDHolder<?>> pcollections = new LinkedHashMap<>();
-  private final Set<RDDHolder<?>> leafRdds = new LinkedHashSet<>();
-  private final Set<PValue> multireads = new LinkedHashSet<>();
+  private final Pipeline pipeline;
+  private long timeout;
```

Comment:
I can only make it final if we give up the constructor chaining, meaning duplication between the two constructors. Is this what we want?